### DISCUSSION TITLE
Allocation target behaviour

### DISCUSSION THREAD

**Discussion by norangebit** [ananthakumaran/paisa#122] [Q&A]
Why the current value sum is different that 100% in the allocation target screen?

<img width="163" alt="Allocation Target" src="https://github.com/ananthakumaran/paisa/assets/4928896/29c93ba0-0d74-4710-b462-d109aece7023">

**Comment by ananthakumaran** [comment]
The current value is calculated as the percentage of total assets. If you haven't included some of your assets account as part of any target group, the total current might not add up to 100. Let me know if that's the case. I will also add a new warning on the doctor page to show the list of asset accounts not included in any allocation target.
